sdk/avalon_sdk/fabric/fabric_wrapper.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from os import environ
+from os.path import exists, realpath
+import json
+import random
+
+from avalon_sdk.fabric import base
+from avalon_sdk.fabric import tx_committer
+from avalon_sdk.fabric import event_listener
+from hfc.protos.common import common_pb2 as common
+
+logging.basicConfig(
+    format='%(asctime)s - %(levelname)s - %(message)s', level=logging.INFO)
+
+
+class FabricWrapper():
+    """
+    Fabric wrapper class to interact with fabric blockchain
+    It provides wrapper functions to invoke and query chain code.
+    """
+
+    def __init__(self, config):
+        """
+        Constructor to initialize wrapper with required parameter
+        Params:
+            config is dict containing parameters for fabric.
+            These parameters are read from toml file.
+        """
+        if self.__validate(config):
+            # Read network file path from fabric toml file.
+            with open(self.__network_conf_file, 'r') as profile:
+                self.__network_config = json.load(profile)
+            self.__channel_name = config["fabric"]["channel_name"]
+            self.__orgname = base.get_net_info(self.__network_config,
+                                               'client', 'organization')
+            logging.info("Org name choose: {}".format(self.__orgname))
+            self.__peername = random.choice(base.get_net_info(
+                self.__network_config, 'organizations', self.__orgname,
+                'peers'))
+            # Get a tx commiter
```

Comment:
Change "commiter" to "committer". Fabric misspelled committer here in the object, but we do not have to in the comment.

Reply:
Done